ros2cli/ros2cli/node/direct.py

```diff
@@ -53,10 +54,31 @@ def timer_callback():
     def __enter__(self):
         return self
 
+    # TODO(hidmic): generalize/standardize rclpy graph API
+    #               to not have to make a special case for
+    #               rclpy.action
+    def get_action_names_and_types(self):
+        return rclpy.action.get_action_names_and_types(self.node)
+
+    def get_action_client_names_and_types_by_node(
+        self,
+        remote_node_name,
+        remote_node_namespace
```

Comment:
Nitpick: how about keeping all these arguments on one line?

Same a few lines below.

Reply:
Huh, yeah, it gets close to a 100 characters long but not quite. Done in https://github.com/ros2/ros2cli/pull/421/commits/e3eee5c56a2faef4664f54289df965f94448bb13.

Reply:
I wasn't asking to keep the whole function signature in one line - only the arguments:

```
def get_action_client_names_and_types_by_node(
        self, remote_node_name, remote_node_namespace
):
```

Reply:
Do you have a strong opinion on this regard? It does comply with `flake8` as-is now, I over-indented it initially.

Reply:
If it passes it can stay as is.